snap/snapcraft.yaml

```diff
@@ -0,0 +1,23 @@
+name: electrum
+version: master
+summary: Bitcoin thin client
+description: |
+  Lightweight Bitcoin client
+
+grade: devel # must be 'stable' to release into candidate/stable channels
```

Comment:
Just as a safeguard during the initial tests. Once it's move to the stable channel in the store it will be installable by millions of users of Ubuntu who can find it in the software center. I'm usually cautious to make sure that there is a commit that marks when the snap is stable. But this is not strictly necessary.